pex/pex_bootstrapper.py

```diff
@@ -116,12 +116,12 @@ def maybe_reexec_pex(compatibility_constraints=None):
 
   current_interpreter = PythonInterpreter.get()
 
-  # NB: Used only for debugging and tests.
-  pex_exec_chain = []
+  # NB: Used only for tests.
   if '_PEX_EXEC_CHAIN' in os.environ:
```

Comment:
I feel slightly less dirty with this tweak. Now user code will never see `_PEX_EXEC_CHAIN` in the environment in production.

Reply:
Was this used somehow before? Confused about the "now" verbage.